admin/assets/javascripts/global-scripts.js

```diff
@@ -154,6 +153,7 @@ $(function () {
      * @param status
      */
     toggle: function (status) {
+      stickyBannerPositioning()
```

Comment:
was thinking alternative naming for this convention.
how about adjustStickyBanner?
From what I noticed in this file the general approach is verb first.

Reply:
I think noun-verb naming is correct

Reply:
@jon-shipley @GeorgeChatzigiannis happy if I rename it adjustStickyBanner? Please confirm.